app/src/main/java/org/openhdp/hdt/data/StopwatchRepository.kt

```diff
@@ -13,5 +13,10 @@ class StopwatchRepository @Inject constructor(
     val stopwatchWithTimestampsDAO: StopwatchWithTimestampsDAO,
     val timestampDAO: TimestampDAO
 ) {
-    // TODO(Add all functions from DAOs)
+
+
+    suspend fun stopwatches() = stopwatchDAO.getAllStopwatchesInOrder()
+
+    suspend fun timestamps(stopwatchId: String) = timestampDAO.getTimestampsFrom(stopwatchId)
+
```

Comment:
OK, but there are more used functions from DAOs. But we can do it in the next version.